SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -19,7 +19,8 @@ public struct MessageBarConfig {
   
   public init(errorColor: UIColor = UIColor.red, successColor: UIColor = .green, infoColor: UIColor = .blue,
               titleColor: UIColor = .white, messageColor: UIColor = .white, statusBarHidden: Bool = false,
-              successIcon: UIImage? = nil, infoIcon: UIImage? = nil, errorIcon: UIImage? = nil) {
+              successIcon: UIImage? = nil, infoIcon: UIImage? = nil, errorIcon: UIImage? = nil,
+              languageDirection: NSLocale.LanguageDirection = .leftToRight) {
```

Comment:
Also what do you think about calling it `forcedLanguageDirection` or similar, essentially that's what it's doing by ignoring the automatic choice